plugin/src/main/resources/io/jenkins/plugins/analysis/warnings/groovy/ParserConfiguration/config.jelly

```diff
@@ -7,10 +7,13 @@
         <f:entry title="${%Parsers}" description="${%title.description}">
           <i:repeatable field="parsers"/>
         </f:entry>
+        <f:entry field="consoleLogScanningPermitted" title="${%consoleLogScanningPermitted.title}" description="${%consoleLogScanningPermitted.description}">
+          <f:checkbox/>
+        </f:entry>
       </j:when>
       <j:otherwise>
         <f:block>
-          You have no permission to edit this section (required permission: RUN_SCRIPTS).
+          You have no permission to edit this section (required permission: ADMINISTER).
```

Comment:
FYI the underlying code checks for Jenkins.ADMINISTER - there's no check for RUN_SCRIPTS.

Reply:
Yes, this has been replaced in Jenkins core recently and I forgot to change the documentation as well.